mne/source_estimate.py

```diff
@@ -3297,6 +3297,8 @@ def extract_label_time_course(stcs, labels, src, mode='mean_flip',
         stcs = [stcs]
         return_several = False
         return_generator = False
+    elif isinstance(stcs, MixedSourceEstimate):
+        stcs = [stcs]
```

Comment:
I don't think this should be part of this PR? (Also I think you want `return_several = return_generator = False` for this when it is eventually part of a PR.)

Reply:
+1